docs/upgrading.md

```diff
@@ -0,0 +1,64 @@
+# Upgrading Contour to 0.15
```

Comment:
Should we make this more generic? Presumably, this applies to all versions above 15.

Reply:
The idea from me was that we will update this document each release, putting the title like that is to remind us to check.